### DISCUSSION TITLE
do you think it is better to talk about ideavim bugs here or the youtrack intelij forum

### DISCUSSION THREAD

**Discussion by KevinBaileyCrum** [JetBrains/ideavim#636] [General]
im never sure what area to put idea vim issues / bugs especially regarding the handful of ideavim issues that have been happening in light of the intelij updates

**Comment by AlexPl292** [comment]
Hi there. Please use [YouTrack](https://youtrack.jetbrains.com/issues/VIM). There is also an action "Create an Issue" from the status bar icon.

I opened GitHub discussions a few years ago and expected that it will be convenient for, well, discussions. [Example](https://github.com/JetBrains/ideavim/discussions/303). However, it's now used for reporting bugs sometimes and I'm not completely sure how to prevent it. Anyway, if I see some real bug reported on GH Discussions, I create a ticket on YouTrack for it.

**Comment by KevinBaileyCrum** [comment]
thanks yes i was never sure if ideavim bugs where for jetbrains/youtrack or here.  Ill open my youtrack ticket when i get time (the new update has interesting behavior when closing tabs in a split.  of course this happens after fixing the :q bug haha)
